features/step_definitions/attachment_steps.rb

```diff
@@ -123,3 +123,28 @@
 Then(/^I should see the html attachment body "(.*?)"$/) do |attachment_body|
   assert page.has_content?(attachment_body)
 end
+
+Given(/^the publication "(.*?)" has an html attachment "(.*?)" with the contact address "(.*?)"$/) do |publication_title, attachment_title, contact_address|
+  publication = Publication.find_by(title: publication_title)
+  create(:html_attachment, attachable: publication, title: attachment_title, meta_data_contact_address: contact_address)
+end
+
+When /^I preview the html attachment "(.*?)" in print mode$/ do |attachment_title|
+  html_attachment = HtmlAttachment.find_by(title: attachment_title)
+  publication = html_attachment.attachable
+  visit publication_html_attachment_path(publication_id: publication.slug, id: html_attachment.slug, medium: "print")
+end
+
+Then /^I should see the metadata page with contact address "(.*?)"$/ do |contact_address|
```

Comment:
You could probably combine this `Then` with the previous `When` so they're more obviously connected.  `Then previewing the html attachment "..." in print mode includes the contact address "..." and isbn "..."` - something like that?  I reckon it'd be good to include checking for the isbn too as it's also a dynamic part of that page.